docs/index.md

```diff
@@ -976,7 +976,7 @@ function importAll(r) {
   r.keys().forEach(r)
 }
 
-importAll(require.context("../components", true, /_component\.js$/))
+importAll(require.context("../components", true, /[_\/]component.js$/))
```

Comment:
I guess this unrelated change should be part of another PR.

Also, why removing the `\` in `component\.js`? It's useful to escape the dot that otherwise means "any character".